### DISCUSSION TITLE
Enabling opacity is vanishing my image on Paraview

### DISCUSSION THREAD

**Discussion by mdmuntaha** [idaholab/moose#25042] [Q&A General]
Hi, Can anyone tell why enabling opacity vanishing the image on paraview? Please see the attached image. 
![MicrosoftTeams-image](https://github.com/idaholab/moose/assets/35989424/91645a54-5f67-40ee-8d97-66be871b3b37)

**Comment by GiudGiud** [comment]
Hello

you probably can ask this on the paraview forum to get an answer quicker. 

Guillaume
